frex/models/constraint_solution_section_set.py

```diff
@@ -0,0 +1,14 @@
+from typing import Tuple, Dict
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+from frex.models import Candidate, ConstraintSolutionSection
+
+
+@dataclass_json
+@dataclass
+class ConstraintSolutionSectionSet:
```

Comment:
is there a better name for describing "SolutionSectionSet"s? and "Section"s in general for that matter.